Packs/VirusTotal/Integrations/VirusTotalV3/VirusTotalV3_description.md

```diff
@@ -0,0 +1,4 @@
+### Authorization:
+Your API key can be found in your VirusTotal account user menu:  
+![](https://files.readme.io/ddeb298-Screen_Shot_2019-10-17_at_3.17.04_PM.png)  
+Your API key carries all your privileges, so keep it secure and don't share it with anyone.
```

Comment:
please add here what needs to be set for the username password from versions before 6.1